proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1198,20 +1198,21 @@ private ITable astTableLink(AST tableAST) {
   }
 
   private SymbolScopeSuper classStateSuper(JPNode classNode, String className) throws RefactorException {
-    LOG.trace("Entering classStateSuper {} {}", classNode, className);
+    return null;
+    /* LOG.trace("Entering classStateSuper {} {}", classNode, className);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)